src/inputBackend/input/DevInput.cpp

```diff
@@ -206,6 +209,34 @@ void DevInput<PointerCallbacks>::onEvent(const input_event& event)
 	}
 }
 
+int setBound(int fd, int axis, uint32_t value)
+{
+	struct input_absinfo abs_info;
+	if (ioctl(fd, EVIOCGABS(axis), &abs_info))
+	{
+		return -1;
+	}
+	abs_info.minimum = 0;
+	abs_info.maximum = value;
+	return ioctl(fd, EVIOCSABS(axis), &abs_info);
+}
+
+void DevInput<PointerCallbacks>::setCallbacks(const PointerCallbacks& callbacks)
+{
+	DevInputCbk<PointerCallbacks>::setCallbacks(callbacks);
+	if (mCallbacks.moveAbsolute)
+	{
+		if (mWidth && setBound(mFd, ABS_X, mWidth))
```

Comment:
Existed code must be modified to handle the possible exceptions 
maybe I am wrong but the following call belongs to the inputDevice (see InputRingBuffer::InputRingBuffe)
mPointer->setCallbacks({
				bind(&InputRingBuffer::onMoveRel, this, _1, _2, _3),
				bind(&InputRingBuffer::onMoveAbs, this, _1, _2, _3),
				bind(&InputRingBuffer::onButton, this, _1, _2),